src/main/java/com/example/FootballManager_back_end/Controller/BaseTeamController.java

```diff
@@ -0,0 +1,48 @@
+package com.example.FootballManager_back_end.Controller;
+
+import com.example.FootballManager_back_end.Dto.BaseTeamDTO;
+import com.example.FootballManager_back_end.Service.BaseTeamService;
+import jakarta.validation.Valid;
+import lombok.AllArgsConstructor;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.net.URI;
+import java.util.List;
+
+@RestController
+@AllArgsConstructor
+@RequestMapping("/base-team")
```

Comment:
rename the @RequestMapping("/base-team") to @RequestMapping("/api/v1/base-team")